test/types/records/generic/generic-with-defaults-q-problem.chpl

```diff
@@ -7,6 +7,6 @@ record K {
 }
 
 {
-  writeln(isGeneric(K));
+  writeln(isGeneric(K(?)));
```

Comment:
If you added a pragma that permits a type to be passed in without a `(?)`, this may be a routine to apply it to?